monkey/infection_monkey/utils/linux/hidden_files.py

```diff
@@ -0,0 +1,35 @@
+HIDDEN_FILE = '/var/tmp/.monkey-hidden-file'
+HIDDEN_FOLDER = '/var/tmp/.monkey-hidden-folder'
```

Comment:
Why are we touching /var/tmp? This artifact won't be deleted at the end of a run.
Also writing to /var will always require root permissions.